eth/CMakeLists.txt

```diff
@@ -1,16 +1,36 @@
 aux_source_directory(. SRC_LIST)
```

Comment:
Also a note: this is also bad. CMake suggests to specify file list manually, but if we want to keep it dynamic `file(GLOB)` is better.